src/main/rx/rx.c

```diff
@@ -680,7 +681,7 @@ void updateRSSI(timeUs_t currentTimeUs)
 
 uint16_t getRssi(void)
 {
-    return (rxConfig()->rssi_scale / 100.0f) * rssi;
+    return ((rxConfig()->rssi_scale / 100.0f) * rssi) + (rxConfig()->rssi_offset * RSSI_OFFSET_SCALING);
```

Comment:
Why do we have so many redundant parentheses?

Reply:
Good point. I added them because `-Wpedantic` was throwing warnings at some stage - turns out the final result doesn't throw warnings any more.

Reply:
My guess is we had macros in use here before PGs, which could indeed cause warnings.